### PR TITLE
Lina development

### PR DESCRIPTION
**Pull request by linatodi**
Landing page and Sign in Frontend

### CODE REVIEW

File: app/controllers/TeamBinary.sublime-project [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"folders":
+	[
+		{
+			"path": "/Users/linatorres/Documents/Team-Binary-2015"
+		}
+	]
+}
```
Comment (sebasoga) [inline app/controllers/TeamBinary.sublime-project]: This file is specific t your text editor and shouldn't really be here, if need to keep this file, yo can add it to the the `.gitignore` file.

File: app/controllers/TeamBinary.sublime-workspace [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+		],
+		"width": 0.0
+	},
+	"show_minimap": true,
+	"show_open_files": false,
+	"show_tabs": true,
+	"side_bar_visible": true,
+	"side_bar_width": 289.0,
+	"status_bar_visible": true
+}
```
Comment (sebasoga) [inline app/controllers/TeamBinary.sublime-workspace]: Same with this file, you can add a line with `*.sublime-project` to the `.gitignore` file so that these files are not tracked by git.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
...
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user
+
+  private
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
 end
```
Comment (sebasoga) [inline app/controllers/application_controller.rb]: Since authentication is not in place yet, I don't think we need this to build the landing page.

File: app/controllers/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    flash[:success] = 'Logged out!'
+    redirect_to root_path
+  end
+
+  private
+
+  def user_params
+    params.require(:session).permit(:username)
+  end
+end
```
Comment (sebasoga) [inline app/controllers/sessions_controller.rb]: I think this doesn't belong in this pull request, the idea here is only to have a basic landing page.

File: app/models/user.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  attr_accessor :password
+end
```
Comment (sebasoga) [inline app/models/user.rb]: Same for the `UserHelper` and this `User` model, I think this probably belong to the feature Angela is working on to use OAuth2 for authentication.

**Comment by sebasoga** [comment]
@linatodi this is a good start, just a few notes:
1. It seems like you're working off a branch that includes @angela4713's work which is why this includes a lot of unrelated changes, you should probably rebase your branch against `master.
2. Don't worry about sessions/user authenticate yet, since they are not in place. If you need to include sign in/out links you can leave add them as placeholders and make them not point anywhere while we get authentication in place.

**Comment by ammancilla** [comment]
@sebasoga @linatodi Another thing we should consider here is adding bootstrap to the project through the asset pipeline using the gem: https://github.com/twbs/bootstrap-sass

**Comment by sebasoga** [comment]
+1 for @ammancilla's suggestion.

**Comment by ammancilla** [comment]
Since Lina did the changes and started working this in another branch, I'm closing this PR.
